ui/shared/AccountSelector.qml

```diff
@@ -179,7 +208,7 @@ Item {
                 anchors.fill: itemContainer
                 hoverEnabled: true
                 onEntered: {
-                    itemContainer.bgColor = Style.current.lightGrey
+                    itemContainer.bgColor = Style.current.grey // change to lightGrey after https://github.com/status-im/nim-status-client/pull/698 is merged
```

Comment:
Assumption on my part:
```suggestion
                    itemContainer.bgColor = Style.current.secondaryBackground
```

Reply:
Agreed, I will address this in an updated https://github.com/status-im/nim-status-client/pull/698.